portafly/portafly_api.yml

```
@@ -74,6 +149,83 @@ components:
         org_name: Umbrella Corp.
         admin_name: Oswell E. Spencer
         apps_count: 3
+    User:
+      title: Root Type for UserData
+      description: A simplified format for a user data.
+      type: object
+      properties:
+        id:
+          description: The user ID number
+          type: integer
+        name:
+          description: The name of the user
+          type: string
+        email:
+          description: The email of the user
+          type: string
+        role:
+          description: The role of the user
+          type: string
+        is_current_user:
+          description: Is current user logged in?
+          type: boolean
+        can_see_users:
+          description: Can the user see other users?
+          type: boolean
+        is_authenticated_through_sso:
+          description: Is the user authenticated through SSO?
+          type: boolean
+        can_delete_user:
+          description: Can current user delete other users?
+          type: boolean
+        can_invite_user:
+          description: Can the current user invite users?
+          type: boolean
+        permissions:
+          description: List of user permissions
+          type: array
+          items:
+            type: string
+    Invitation:
+      title: Root Type for InvitationData
+      description: A simplified format for an invitation data.
+      type: object
+      properties:
+        id:
+          description: The invitation ID number
+          type: integer
+        email:
+          description: The email of the user invited
+          type: string
+        sent_at:
+          format: date-time
+          description: Date when the invitation was sent
+          type: string
+        is_accepted:
+          description: Is the invitation accepted?
+          type: boolean
+        accepted_at:
+          format: date-time
+          description: Date when the invitation was accepted
+          type: string
+      example:
+        id: 2
+        email: stan@southpark.com
+        sent_at: 2019-10-18T05:13:26Z
+        accepted: true
+        accepted_at: 2019-10-19T05:13:26Z
```

I would name this "generic" examples so that they can be reused eventually. For instance it can be reused already in `InvitationNew`.
```suggestion
        $ref: '#/components/examples/InvitationExample'
```